Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "frequenz-microgrid-component-graph-rs"
```

Comment:
Do we want to keep the `-rs` suffix in the package name? At least in Python we remove the suffix.

Reply:
done.